File: Tipos de Dados/Registros/RegistroPessoa.pas
//Pascal

program RegistroPessoa;

type
    Pessoa = record
        nome: string;
        idade: integer;
        altura: real;
    end;

var
    p: Pessoa;

begin
    // Atribuindo valores
    p.nome := 'Ana';
    p.idade := 28;
    p.altura := 1.65;

    // Exibindo valores
    writeln('Nome: ', p.nome);
    writeln('Idade: ', p.idade);
    writeln('Altura: ', p.altura:0:2, ' m');
end.

File: Tipos de Dados/Registros/RegistroPessoa.cs
//C Sharp (C#)
//Aqui o registro é chamado de Estrutura (struct)

using System;

struct Pessoa {
    public string Nome;
    public int Idade;
    public float Altura;
}

class RegistroPessoa {
    static void Main() {
        Pessoa pessoa;

        // Atribuindo valores
        pessoa.Nome = "Carlos";
        pessoa.Idade = 40;
        pessoa.Altura = 1.80f;

        // Exibindo valores
        Console.WriteLine("Nome: " + pessoa.Nome);
        Console.WriteLine("Idade: " + pessoa.Idade);
        Console.WriteLine("Altura: " + pessoa.Altura + " m");
    }
}
